resolwe_api/resolwe.py

```diff
@@ -1,5 +1,5 @@
 """Resolwe"""
-from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Did you find out what exactly causes the crash in case `unicode_literals` are used?